qiskit/pulse/builder.py

```diff
@@ -1699,8 +1709,25 @@ def call(target: Union[circuit.QuantumCircuit, Schedule]):
                 pulse.call(sched)
                 pulse.call(qc)
 
+    This function can optionally take parameter dictionary with the parameterized target program.
+
+    .. jupyter-execute::
+
+        from qiskit import circuit, pulse
+
+        amp = circuit.Parameter('amp')
+
+        with pulse.build() as subroutine:
+            pulse.play(pulse.Gaussian(160, amp, 40), pulse.DriveChannel(0))
+
+        with pulse.build() as main_prog:
+            pulse.call(subroutine, {amp: 0.1})
+            pulse.call(subroutine, {amp: 0.3})
```

Comment:
Would it be possible to do
`pulse.call(subroutine, amp=0.1)`
or no because of the way parameters work?